layers/+distributions/spacemacs-base/local/centered-buffer-mode/centered-buffer-mode.el

```diff
@@ -52,7 +52,6 @@ that differed modifications won't cause an overflow."
   "Minor mode to center buffer in its window."
   :init-value nil
   :group 'editing-basics
-  :lighter " -B-"
```

Comment:
I removed the lighter because it's pretty obvious when the mode is on. 
No lighter no problem with its name :smile: